### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by msh855** [hootnot/oanda-api-v20#213] [General]
Hi, 

I am very new to your library. I went through the documentation, but it is very hard to understand how to use the library. 

For example, I manage to place an order based on this script: 

```

 client = oandapyV20.API(access_token=api, environment="practice")
    order = {"order": {"instrument": cross,
                       "units": str(market_order_units),
                       "type": "MARKET",
                       "positionFill": "DEFAULT"}}
    r = orders.OrderCreate(accountID=account_id, data=order)
    client.request(r)
```

But I have no idea what my options are for `"type"`, for instance, how do I place a limit order.  How do I know what instruments I can use. It seems the strings you are using do not follow the conventions, for example, instead of  `USD/JPY`, the API takes `USD_JPY` as input. Also, what are the strings for stocks or bonds, or it is only for FX. 

I think people will highly appreciate a more user-friendly documentation if it is possible.

**Comment by hootnot** [comment]
Hi,

The oandapyV20 library is an API wrapper for the OANDA V20 API. So, it is only to trade at OANDA. Request the instruments list and you will know what you can trade. 

OANDA uses symbols as  USD_JPY.

Regarding documentation: check https://developer.oanda.com/rest-live-v20/introduction/, you will find full documentation of their V20 API.

Also check: https://oanda-api-v20.readthedocs.io/en/latest/oandapyV20.definitions.orders.html

**Reply by msh855** [reply]
I see. 

I still see some good potential to improve the documentation of the wrapper so users do not go back and forth and get confused with the syntax. 

In any case, thanks for the links. I would have a look.
